### PR TITLE
Add advanced client for crypto data

### PR DESCRIPTION
**Pull request by zhaohuxing**

### CODE REVIEW

File: client/crypto/crypto.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import (
+	"errors"
+
+	"github.com/yunify/qingstor-sdk-go/service"
+)
+
+// Algorithm is the set of all unsigned 64-bit integers.
+// Range: 0 through 18446744073709551615.
+type Algorithm uint64
+
```
Comment (pengsrc) [inline client/crypto/crypto.go:11]: Need to support 18446744073709551615 kinds of crypto algorithm?

Are you kidding me?

**Review by pengsrc — CHANGES_REQUESTED** [review]

**Comment by pengsrc** [comment]
We have go `Upload` client [here](https://github.com/yunify/qingstor-sdk-go/blob/master/client/upload/upload_client.go), that's a convenient way to upload large files, no more `UploadMultipart()` in advanced clients.

File: client/crypto/crypto.go (hunk 1 above)
Comment (zhaohuxing) [reply client/crypto/crypto.go:11]: // AES256 represents aes56 algorithm
 AES256 Algorithm = 1 << iota
There are only 128 types of algorithms
OK, I modify `uint64` to `uint8`.

**Comment by zhaohuxing** [comment]
Updated 20170930 1415

**Comment by pengsrc** [comment]
> Updated 20170930 1415

What dose it means?

**Comment by pengsrc** [comment]
The `crypto` is not a verb, and also verb words after `for` should be in the present continuous tense.

**Comment by zhaohuxing** [comment]
Thanks, I modified the description of the commit.

**Pull request closed without merging**